workflow.md

```diff
@@ -100,8 +100,8 @@ The current `instance` is accessed in the same way as the `context`.
 import pyblish.api
 
 class MyPlugin(pyblish.api.Plugin):
-    def process(self, context):
-        self.log.info("Publishing from {cwd}".format(**context.data))

```

Comment:
This is actually intentional, it's to show the `.data` member of the context, and that it contains a key called `cwd`. Maybe a different example would be better if it isn't clear enough. Do you think you could change it to something more obvious, or change it back?